### PR TITLE
Switch to using easyocr

### PR DESCRIPTION
**Pull request by cmolodowitch**
Modified OCR.py to use EasyOCR instead of keras-ocr, seems slightly more accurate, though full test is pending.  Updated requirements.txt accordingly.

I installed django locally and tried running the branch and it worked for me!  Since you already have PyTorch/torchvision installed for the other models, the only thing you should need to add to your local environment is easyocr.

### CODE REVIEW

**Comment by cmolodowitch** [comment]
EasyOCR accuracy on "front" Split SPL images 31.6% vs keras-ocr accuracy 26.9%, better to use EasyOCR
